widgetssdk/src/main/java/com/glia/widgets/core/notification/domain/CallNotificationUseCase.kt

```diff
@@ -0,0 +1,65 @@
+package com.glia.widgets.core.notification.domain
+
+import com.glia.androidsdk.comms.MediaDirection
+import com.glia.androidsdk.comms.OperatorMediaState
+import com.glia.androidsdk.comms.VisitorMediaState
+import com.glia.widgets.core.notification.device.INotificationManager
+
+internal class CallNotificationUseCase(
+    private val notificationManager: INotificationManager
+) {
+    operator fun invoke(visitorMedia: VisitorMediaState? = null, operatorMedia: OperatorMediaState? = null) {
+        val audio: MediaDirection
+        val video: MediaDirection
+        try {
+            audio = getAudioDirection(visitorMedia, operatorMedia)
+            video = getVideoDirection(visitorMedia, operatorMedia)
+        } catch (error: IllegalStateException) {
+            // One of the impossible scenario was detected during parsing of the states which is not supported by current GliaHub
+            // TODO: should it throw exception or return error or just silently log warning
```

Comment:
Regarding this TODO. I would vote for a silent error log. We should notice it in Sentry and take action.

Reply:
Oh, right!
Forgot about this one. Thank you!
I will add a silent error log. But unless something changed lately in Widgets SDK we don't yet send the errors to the server. Only the Core SDK does that

Reply:
Fixed in 5716d01